feature/lldp/otg_tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go

```diff
@@ -143,7 +144,10 @@ func configureDUT(t *testing.T, name string, lldpEnabled bool) (*ondatra.DUTDevi
 	gnmi.Replace(t, node, lldp.Enabled().Config(), lldpEnabled)
 
 	if lldpEnabled {
-		gnmi.Replace(t, node, lldp.Interface(p.Name()).Enabled().Config(), lldpEnabled)
+		gnmi.Replace(t, node, lldp.Interface(p.Name()).Config(), &oc.Lldp_Interface{
+			Name:    ygot.String(p.Name()),
```

Comment:
This looks unrelated to NI

Reply:
Yes, but this a similar problem that needs to be fixed. Interface name must be configured before configuring enabled. Below are similar. Should I submit these changes in separate PRs?

Reply:
Yes please. Thank you for the contribution.

Reply:
Done